### PR TITLE
[DNM] fit for go-hbase refactor

### PR DESCRIPTION
**Pull request by morefreeze**
I extract function signature which be modified from https://github.com/pingcap/go-hbase/pull/5. All of them just need add a `error` value to receive, but don't handle the error.
We `make` it pass first.
PR from tidb pingcap/tidb#703
@c4pt0r @qiuyesuifeng PTAL

### CODE REVIEW

File: util_test.go [hunk 1]
```diff
@@ -50,7 +50,11 @@ func createHBaseClient() (hbase.HBaseClient, error) {
 }
 
 func createNewTableAndDropOldTable(cli hbase.HBaseClient, tblName string, family string, splits [][]byte) error {
-	if cli.TableExists(tblName) {
+	b, err := cli.TableExists(tblName)
```
Comment (qiuyesuifeng) [inline util_test.go]: Seems can directly use dropTable?

File: themis_txn.go [hunk 2]
```diff
@@ -254,7 +254,12 @@ func (txn *themisTxn) batchCommitSecondary(wait bool) {
 func (txn *themisTxn) groupByRegion() map[string]map[string]*rowMutation {
 	rsRowMap := make(map[string]map[string]*rowMutation)
 	for _, rm := range txn.secondaryRows {
-		key := getBatchGroupKey(txn.client.LocateRegion(rm.tbl, rm.row, true), string(rm.tbl))
+		region, err := txn.client.LocateRegion(rm.tbl, rm.row, true)
```
Comment (qiuyesuifeng) [inline themis_txn.go]: If locate region error, should we return error and retry call groupByRegion?

File: util_test.go (hunk 1 above)
Comment (morefreeze) [inline util_test.go]: yep, I'll fix it

File: themis_txn.go (hunk 2 above)
Comment (morefreeze) [inline themis_txn.go]: @c4pt0r

File: mutation_cache.go [hunk 3]
```diff
@@ -99,7 +99,9 @@ func (r *rowMutation) getColumns() []hbase.Column {
 	var ret []hbase.Column
 	for k, _ := range r.mutations {
 		c := &hbase.Column{}
-		c.ParseFromString(k)
+		// TODO: handle error, now just ignore
+		if err := c.ParseFromString(k); err != nil {
```
Comment (c4pt0r) [inline mutation_cache.go]: log.Error

File: themis_primary_lock.go [hunk 4]
```diff
@@ -53,8 +55,12 @@ func (l *themisPrimaryLock) Encode() []byte {
 	binary.Write(buf, binary.BigEndian, int32(len(l.secondaries)))
 	for k, v := range l.secondaries {
 		c := &hbase.ColumnCoordinate{}
-		c.ParseFromString(k)
-		c.Write(buf)
+		// TODO: handle error, now just ignore
+		if err := c.ParseFromString(k); err != nil {
```
Comment (c4pt0r) [inline themis_primary_lock.go]: return it

**Comment by morefreeze** [comment]
@c4pt0r @qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
@c4pt0r  PTAL

**Comment by c4pt0r** [comment]
LGTM

**Pull request merged**